server/apps/archive/commands.py

```diff
@@ -50,11 +61,68 @@ def _get_query_for_expired_items(self, now):
         query = {
             '$and': [
                 {'expiry': {'$lte': now}},
-                {'state': 'spiked'}
+                {'state': CONTENT_STATE.SPIKED}
             ]
         }
 
         return query
 
 
+class UpdateOverdueScheduledContent(superdesk.Command):
+    """
+    Updates the overdue scheduled stories
+    """
+
+    def run(self):
+        self.update_overdue_scheduled()
+
+    def update_overdue_scheduled(self):
+        """
+        Updates the overdue scheduled content on archive collection.
+        """
+
+        logger.info('Updating overdue scheduled content')
+
+        if is_task_running("archive", "update_overdue_scheduled", UPDATE_OVERDUE_SCHEDULED_DEFAULT):
+            return
+
+        try:
+            now = date_to_str(utcnow())
+            items = get_overdue_scheduled_items(now, ARCHIVE)
+            item_update = {'state': CONTENT_STATE.PUBLISHED}
+
+            for item in items:
+                logger.info('updating overdue scheduled article with id {} and headline {} -- expired on: {} now: {}'.
+                            format(item['_id'], item['headline'], item['publish_schedule'], now))
+
+                superdesk.get_resource_service('archive').patch(item['item_id'], item_update)
+        finally:
+            mark_task_as_not_running("archive", "update_overdue_scheduled")
+
+
+def get_overdue_scheduled_items(expired_date_time, resource, limit=100):
+    """
+    Fetches the overdue scheduled articles from given collection. Overdue Conditions:
+        1.  it should be in 'scheduled' state
+        2.  publish_schedule is less than or equal to expired_date_time
+
+    :param expired_date_time: DateTime that scheduled tate will be checked against
+    :param resource: Name of the resource to check the data from
+    :param limit: Number of return items
+    :return: overdue scheduled articles from published collection
+    """
+
+    logger.info('Get overdue scheduled content from {}'.format(resource))
+    query = {'$and': [
+        {'publish_schedule': {'$lte': expired_date_time}},
+        {ITEM_STATE: CONTENT_STATE.SCHEDULED}
+    ]}
+
+    req = ParsedRequest()
+    req.sort = '_created'
```

Comment:
Sort should be on `_modified` aka the item which is scheduled first instead of `_created`.